cpp/src/cider/exec/nextgen/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set(NEXTGEN_DEPS cider_util gflags)
+set(NEXTGEN_DEPS cider_util ${GFLAGS_LIBRARIES})
```

Comment:
what's the difference?

Reply:
no difference, but prefer to CMake style